### PR TITLE
Service config

### PR DESCRIPTION
**Pull request by felixleo22**
Feature: 

- At the start of each service, it display log the node config without secrets keys

### CODE REVIEW

File: src/apikey/app.js [hunk 1]
```diff
@@ -39,6 +40,7 @@ app.use((error, req, res, next) => res.status(500).json({ message: error.message
 
 app.listen(3000, async () => {
   logger.info('ezunpaywall apikey service listening on 3000');
+  getConfig(true);
```
Comment (oxypomme) [inline src/apikey/app.js:43]: You may want to use env instead of forcing it

File: src/enrich/app.js [hunk 2]
```diff
@@ -53,6 +54,7 @@ app.use((error, req, res, next) => res.status(500).json({ message: error.message
 
 app.listen(3000, () => {
   logger.info('ezunpaywall enrich service listening on 3000');
+  getConfig(true);
```
Comment (oxypomme) [inline src/enrich/app.js:57]: You may want to use env instead of forcing it

File: src/health/app.js [hunk 3]
```diff
@@ -27,4 +28,5 @@ app.use((error, req, res, next) => res.status(500).json({ message: error.message
 
 app.listen(3000, async () => {
   logger.info('ezunpaywall health service listening on 3000');
+  getConfig(true);
```
Comment (oxypomme) [inline src/health/app.js:31]: You may want to use env instead of forcing it

File: src/mail/app.js [hunk 4]
```diff
@@ -37,4 +38,5 @@ app.use((error, req, res, next) => res.status(500).json({ message: error.message
 
 app.listen(3000, () => {
   logger.info('ezunpaywall mail service listening on 3000');
+  getConfig(true);
```
Comment (oxypomme) [inline src/mail/app.js:41]: You may want to use env instead of forcing it

File: src/update/app.js [hunk 5]
```diff
@@ -54,5 +55,6 @@ app.use((error, req, res, next) => res.status(500).json({ message: error.message
 
 app.listen(3000, async () => {
   logger.info('ezunpaywall update service listening on 3000');
+  getConfig(true);
```
Comment (oxypomme) [inline src/update/app.js:58]: You may want to use env instead of forcing it

File: src/graphql/app.js [hunk 6]
```diff
@@ -57,8 +57,10 @@ app.use((error, req, res, next) => res.status(500).json({ message: error.message
 
 app.listen(3000, () => {
   logger.info('ezunpaywall graphQL API listening on 3000');
-  pingElastic();
-  pingRedis();
-  setMetrics();
+  getConfig(true);
```
Comment (oxypomme) [inline src/graphql/app.js:60]: You may want to use env instead of forcing it

**Review by oxypomme — CHANGES_REQUESTED** [review]

**Review by nojhamster — APPROVED** [review]

**Review by oxypomme — APPROVED** [review]

**Pull request merged**